docs/resources/lts_host_group.md

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Log Tank Service (LTS)"
+---
+
+# huaweicloud_lts_host_group
+
+Manages a LTS host group resource within HuaweiCloud.
```

Comment:
```suggestion
Manages an LTS host group resource within HuaweiCloud.
```
The letter L is pronounced `[el]`
And check other places

Reply:
fixed